keras/__init__.py

```diff
@@ -0,0 +1,18 @@
+
+"""
+Keras: Theano-based Deep Learning library
+==================================
+Keras is a minimalist, highly modular neural network library in 
+the spirit of Torch, written in Python / Theano so as not to have 
+to deal with the dearth of ecosystem in Lua. It was developed with 
+a focus on enabling fast experimentation. Being able to go from 
+idea to result with the least possible delay is key to doing 
+good research.
+
+See http://keras.io/
+"""
+import sys
+import re
+import warnings

```

Comment:
They will be useful in the future but for now, it's not needed, so I've removed it. Sorry for the unused imports =)

Usually `sys` is used in the `__init__.py` for looking for the system files, like what `sklearn` does, `nltk` uses `os` functions instead. 

`re` was used by `sklearn` to do string matches and `warnings` was used for deprecated.